socorro/external/es/supersearch.py

```diff
@@ -248,22 +248,50 @@ def get(self, **kwargs):
                 elif param.data_type == 'str' and not param.operator:
                     param.value = [x.lower() for x in param.value]
 
+                # Operators needing wildcards, and the associated value
+                # transformation with said wildcards.
+                operator_wildcards = {
+                    '~': '*%s*',  # contains
+                    '$': '%s*',  # starts with
+                    '^': '*%s'  # ends with
+                }
+                # Operators needing ranges, and the associated Elasticsearch
+                # comparison operator.
+                operator_range = {
+                    '>': 'gt',
+                    '<': 'lt',
+                    '>=': 'gte',
+                    '<=': 'lte',
+                }
+
                 args = {}
                 filter_type = 'term'
                 filter_value = None
+
                 if not param.operator:
                     # contains one of the terms
                     if len(param.value) == 1:
                         val = param.value[0]
+
                         if not isinstance(val, basestring) or (
                             isinstance(val, basestring) and ' ' not in val
                         ):
+                            # There's only one term and no white space, this
+                            # is a simple term filter.
                             filter_value = val
-
-                        # If the term contains white spaces, we want to perform
-                        # a phrase query. Thus we do nothing here and let this
-                        # value be handled later.
+                        else:
+                            # If the term contains white spaces, we want to
+                            # perform a phrase query.
+                            filter_type = 'query'
+                            query = Q(
```

Comment:
Perhaps change to `args = Q(...).to_dict()` if you don't need the `query` variable.